client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx

```diff
@@ -319,35 +239,21 @@ class FrontsCAPISearchInput extends React.Component<
     val: string
   ) {
     const arr = this.state[key];
-    this.setStateAndRunSearch({
+    this.setStateInner({
       ...this.state,
       [key]: arr.includes(val) ? arr.filter(v => v !== val) : arr
     });
   }
 
-  private debouncedRunSearchAndRestartPolling = () => {};
-
-  private setStateAndRunSearch<K extends keyof FrontsCAPISearchInputState>(
-    state: Pick<FrontsCAPISearchInputState, K>
+  private setStateInner<K extends keyof SearchInputState>(
```

Comment:
I *think*, because `K` extends `keyof SearchInputState `, that it can be any subset of keys from `SearchInputState`. Rather if `K` where just equal to `keyof SearchInputState` (rather than extending it) then it would be all of them. At least that's what I assume, and it seems to work!